src/views/LandingView.vue

```diff
@@ -108,33 +127,20 @@ p {
 }
 
 @keyframes float-outer {
-  0%, 100% {
+  0%,
+  100% {
     transform: translateY(0);
   }
   50% {
     transform: translateY(3px);
   }
 }
 
-@keyframes float {
-  0%, 100% {
-    transform: translateY(0);
-  }
-  50% {
-    transform: translateY(-5px);
-  }
-}
-
 .floating-button1 {
```

Comment:
We should aim for more descriptive class names - you usually want to avoid things like "button-1" and "button-2" because other developers will need to go find out what they do.